app/_includes/info_box/sections/topologies.html

```diff
@@ -1,8 +1,19 @@
 <div class="flex flex-col gap-2  w-full">
-  <div class="text-primary font-medium">Supported Topologies</div>
+  <div class="text-primary font-medium">Supported Gateway Topologies</div>
```

Comment:
```suggestion
  <div class="text-primary font-medium">Gateway Topologies</div>
```
Just a suggestion if you want to shorten it. FWIW, I think "Supported Gateway Topologies" works and doesn't seem too long from my small laptop screen :)